tests/queries/0_stateless/01722_long_brotli_http_compression_json_format.reference

```diff
@@ -19,5 +21,3 @@
 
 	"rows": 1000000,
 
-	"rows_before_limit_at_least": 1048080,
```

Comment:
Why is this missing from the output? It is not obvious for me.

Reply:
Because `NumbersRange` will not generate numbers out of range, so there is no need to add `LimitTransform` after `scan`.